### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by parthi2929** [icaoberg/latex2png#2]
Hi

Can you please make it a chrome extension?

**Comment by icaoberg** [comment]
Could you elaborate? What exactly do you need?

**Comment by parthi2929** [comment]
The specific context is, I am unable to use latex in github issues. I know github not yet supporst so looking for workarounds like this.  I would like to have a pop up (or automatic), which could convert my latex (inline and display mode), to PNGs so the writing is seamless. If it could be done automatically that would be great, if not at least as a pop up  so I could have pngs without leaving github page. Currently I am having to switch websites (from here to latex2png and back).

**Comment by icaoberg** [comment]
I don't think that is feasible with this project. However, [MimeTex](http://www.forkosh.com/mimetex.html) might be useful.
